storage/src/qmdb/any/unordered/fixed.rs

```diff
@@ -45,7 +46,14 @@ impl<E: Storage + Clock + Metrics, K: Array, V: FixedValue, H: Hasher, T: Transl
         callback: impl FnMut(bool, Option<Location>),
     ) -> Result<Self, Error> {
         let translator = cfg.translator.clone();
-        let log = init_fixed_authenticated_log(context.clone(), cfg).await?;
+        let mut log = init_fixed_authenticated_log(context.clone(), cfg).await?;
+        if log.size() == 0 {
+            warn!("Authenticated log is empty, initializing new db");
```

Comment:
nit: should this be `info!`? `warn!` seems to suggest something potentially bad happened, but this is expected.

Reply:
My thinking was in the typical case you're using an existing DB, and initializing a new DB would be unusual, often meaning you provided a bad config?  It's a mistake I make often anyway!